src/main/java/it/pagopa/transactions/configurations/AzureStorageConfig.java

```diff
@@ -42,6 +42,24 @@ public QueueAsyncClient transactionActivatedQueueAsyncClient(
         return queueAsyncClient;
     }
 
+    @Bean("transactionRefundQueueAsyncClient")
+    @Qualifier
+    public QueueAsyncClient transactionRefundQueueAsyncClient(
+                                                              @Value(
+                                                                  "${azurestorage.connectionstring}"
+                                                              ) String storageConnectionString,
+                                                              @Value(
+                                                                  "${azurestorage.queues.transactionrefund.name}"
+                                                              ) String queueName
+    ) {
+        QueueAsyncClient queueAsyncClient = new QueueClientBuilder()
+                .connectionString(storageConnectionString)
+                .queueName(queueName)
+                .buildAsyncClient();
+        queueAsyncClient.createIfNotExists().block();
```

Comment:
```suggestion
```
Remove it so that if the queue is not created for an environment the module will fail starting avoiding creating misconfigured queue's.
Please apply it to all queue's into this class

Reply:
Reminder for the future: queue creation to be removed in a separate PR that will include queue creation step before code review tests. See task CHK-1232